### ISSUE TITLE
Installer flagged as virus

### ISSUE THREAD

**Issue by weawer** [brianmiller/phvalheim-client#7]
hey
Are you doing something a bit strange when installing the client? It was flagged by my f-secure installation and by some others over at virusttotal.
![image](https://user-images.githubusercontent.com/1397413/212715270-78977ffa-fbd1-43a4-b2b3-58710cf4daa5.png)

**Comment by brianmiller** [comment]
wtf. No. It's a self-signed installer. It may be time to pay for a signing key... :(

Just make sure you're using the one from this GitHub page and it'll be safe.

**Comment by weawer** [comment]
yeah i dont doubt its safe i just found it odd that it was flagged. It was the most recent release from the github.

**Comment by brianmiller** [comment]
Yeah, I appreciate the heads-up.  I haven't seen this yet.  I'm looking at purchasing a signing key today.  The project is getting enough attention to warrant the need. 👍 

I'm leaving this issue open until the published executables are signed by a publicly trusted key.

**Comment by MasonColoretti** [comment]
any further info on the signing key?

**Comment by brianmiller** [comment]
I need to update my mailing address still to get through the digital identity verification.  I haven't gotten around to it yet.
